egs/ami/s5b/local/nnet3/run_ivector_common.sh

```diff
@@ -188,8 +191,8 @@ if [ $stage -le 8 ]; then
   temp_data_root=${ivectordir}
   utils/data/modify_speaker_info.sh --utts-per-spk-max 2 \
     data/${mic}/${train_set}_sp_hires_comb ${temp_data_root}/${train_set}_sp_hires_comb_max2
-  
-  steps/online/nnet2/extract_ivectors_online.sh --cmd "$train_cmd" --nj $nj \
+
+  steps/online/nnet2/extract_ivectors_online.sh --cmd "$train_cmd" --nj 120 \
```

Comment:
since we generate "$num_cmn_offsets * num_train_frames" i-vectors for training data, I increased nj for extracting i-vector for training data.
I don't want to increase nj for ivector generation for test data, since it may cause problem due to less speakers per job.